src/main/java/com/massivedisaster/bintraydeployautomator/Automator.java

```diff
@@ -20,56 +22,85 @@
      *
      * @param args command line arguments.
      */
-    public static void main(String args[]) {
+    public static void main(String args[]) throws IOException {
 
         ProjectConnection gradleConnection = null;
+        ProgressBar pb = null;
+        String outputFile = null;
         try {
-            Pair<String, String> auth = CommandLineUtils.commandLineArgs(args);
```

Comment:
If the `Pair ` class ins't used anymore, remove it.

Reply:
Pair class is from javafx.util.Pair not an actual class from the project, the import was removed to as you can see above.

Reply:
I had ideia that it was created by us.